.github/workflows/github-release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create GitHub release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
The latest checkout action was updated to [v4](https://github.com/actions/checkout/releases/tag/v4.1.1) a few days ago. And [dependabot](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-supply-chain-security#what-is-dependabot) could be used to keep actions up-to-date.

Reply:
> And [dependabot](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-supply-chain-security#what-is-dependabot) could be used to keep actions up-to-date.

Good point, would you mind to create an issue so we don't lost track of this? Thanks!

Reply:
But I could only create bug report, blank issues are not allowed.